qiskit_experiments/visualization/plotters/iq_plotter.py

```diff
@@ -93,9 +93,14 @@ def expected_supplementary_data_keys(cls) -> List[str]:
                 classifies the provided series data. Must be a subclass of :class:`BaseDiscriminator`.
                 See :attr:`options` for ways to control the generation of the discriminator prediction
                 image.
+            fidelity: A float representing the fidelity of the discrimination reported as
+                :math:`F = 1 - \frac{1}{d}\sum{i\neq j}P(i|j)`. Here, :math:`d` is the number of
+                levels that were discriminated while :math:`P(i|j)` is the probability of measuring
+                outcome :math:`i` given that state :math:`j` was prepared.
```

Comment:
Given that `IQPlotter` doesn't compute the fidelity, would it make sense just to state that `fidelity` is some arbitrary fidelity reported by the analysis?